packages/integration/lib/src/views/post/post_detail_screen.dart

```diff
@@ -109,39 +99,28 @@ class _LMFeedPostDetailScreenState extends State<LMFeedPostDetailScreen> {
   bool isAndroid = LMFeedPlatform.instance.isAndroid();
   bool isWeb = LMFeedPlatform.instance.isWeb();
   double? screenWidth;
+  final _commentBloc = LMFeedCommentBloc.instance();
+  final ValueNotifier<bool> rebuildPostWidget = ValueNotifier(false);
+  LMPostViewData? postData;
+  final TextEditingController _commentController = TextEditingController();
```

Comment:
Why is this controller needed here, and variable for access to comment bloc?

Reply:
unused controller, removed from here.